src/scss/components/_badges.scss

```diff
@@ -49,20 +50,6 @@ $badge-angled-rounded-radius: 2px !default;
             background: none;
         }
 
-        //Both c-badge--light and c-badge--transparent share the same divider styles,
-        //The content '\2022' will output a bullet: •
-        .c-badge--light,
```

Comment:
Don't think these were supposed to be deleted?

Reply:
Yep – they've been moved down onto to [line 125](https://github.com/justeat/fozzie/pull/151/files#diff-f8da4ba2708cb39269c6e11427363480R125) as part of the convo with @sonnyprince to get rid of styling the span's directly.

Didn't need the wrapping modifier then as it has it's own named class.